app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  validates :title, presence: true
+  validates :text, presence: true
+  attr_accessor :comments_counter
+  attr_accessor :likes_counter
+
+  belongs_to :author, foreign_key: 'author_id', class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_author_posts_count
+    author.update(posts_counter: author.posts.count)
+  end
```

Comment:
- [ ] It would be nicer to be calling the `update_author_posts_count` every time a post is created and saved to the database. Make sure the after_save method is called just on top after the validates
```rb
# app/models/post.rb
  after_save :update_author_posts_count

  private

  def update_author_posts_count
    ...
  end

```